godot-core/src/registry.rs

```diff
@@ -6,9 +6,10 @@
 
 #![allow(dead_code)] // FIXME
 
-use crate::obj::*;
+use crate::init::InitLevel;
 use crate::private::as_storage;
 use crate::storage::InstanceStorage;
+use crate::{log, obj::*};
```

Comment:
Prefer flat imports if possible (see also [book](https://godot-rust.github.io/book/gdext/contribute/conventions.html#code-organization)) 🙂

Reply:
oh yeah lol, maybe we could try configuring rustfmt to do this by default, i think it's possible but might be an unstable feature.